tests/testthat/test_family_tax_benefit.R

```diff
@@ -0,0 +1,84 @@
+context("Family tax Benefit")
+
+test_that("Fringe cases", {
+  #no income
+  temp <- data.table(HH_id = c(1, 1, 1), pers_id = 1:3, age = c(35, 30, 0), income = c(0, 0, 0),
+                     in_secondary_school = F, single_parent = F, other_allowance_benefit_or_pension = F, maintenance_income = 0, maintenance_children = 0)
+  expect_equal(family_tax_benefit(temp), 
+               data.table(HH_id = 1, ftbA_incl_supplement = 5412.95, ftbB_incl_supplement = 4339.85))
```

Comment:
Is creating a data table like this in a test ok? Thought it makes it easier to understand.

Reply:
Yes, but use `library(data.table)` within the `test_that`

Reply:
Does it need to go in each test_that or just at the top of the file?